website/src/main/java/com/google/step/data/OrganizationInfo.java

```diff
@@ -65,4 +79,43 @@ public Query getQueryForDuplicates() {
   public static OrganizationInfo createInstanceFrom(HttpServletRequest request) {
     return new OrganizationInfo(getOrgEntityFrom(request));
   }
+
+  public static List<OrganizationInfo> getOrganizationsFrom(CSVReader csvReader) throws IOException {
+    List<OrganizationInfo> organizations = new ArrayList<>();
+    String[] nextRecord = new String[3];
+    int index = 0; 
+    try {
+      while ((nextRecord = csvReader.readNext()) != null) { 
+        try {
+          OrganizationInfo item = new OrganizationInfo(getOrganizationEntityFrom(nextRecord, index++)); 
+            organizations.add(item);
+        } catch (Exception ex) {
+          System.err.println(ex);
+        }
+      }
+    } catch (CsvValidationException ex) {
+      System.err.println(ex);
+    } catch (DeadlineExceededException ex) {
+      return organizations;
+    }
+    return organizations;    
+  }
+
+  /** Detects categories in text using the Language Beta API. */
+  private static List<String> classifyText(String text) throws Exception {
```

Comment:
You have a try block below without a catch statement and declared throws in the function header. It looks ok to me, but would be better if you were more consistent (use either throws or use try catch and log the error message but not mix both)

Reply:
Ah I see, it gets tricky with return statements... keep what you have then sorry